xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -418,10 +418,12 @@
   "Audio.Fields.Artist": {
     "extends": "Item.Fields.Base",
     "items": { "type": "string",
+      "description": "Requesting the (song)genreid/genre, or roleid/role fields will result in increased response times",
       "enum": [ "instrument", "style", "mood", "born", "formed",
                 "description", "genre", "died", "disbanded",
                 "yearsactive", "musicbrainzartistid", "fanart",
-                "thumbnail", "compilationartist", "dateadded" ]
+                "thumbnail", "compilationartist", "dateadded",
+                "roleid", "role", "songgenreid", "songgenre", "isalbumartist"]
```

Comment:
From GetArtists, and it seems to work, but have I made a mistake? Should they also be added to Audio.Details.Artist ?